frontend/src/api/dateHandling.ts

```diff
@@ -82,12 +82,14 @@ function isBetweenDates(startDateStr: string, endDateStr: string, todayDate: Dat
   const startDate = new Date(
     parseInt(startDateStr.slice(0, 4), 10),
     parseInt(startDateStr.slice(4, 6), 10) - 1,
-    parseInt(startDateStr.slice(6, 8), 10)
+    parseInt(startDateStr.slice(6, 8), 10),
+    2
```

Comment:
why/how does this work?

Reply:
Goofy Zeitumstellung.
Aus 2024-05-24:00
Wird 2024-05-23:22